src/views/Jobs/JobResultsTable.vue

```diff
@@ -949,8 +949,20 @@ export default Vue.extend({
                   : null,
               showDiffFVAScore: false
             };
-            // Knockout genes or reactions depending on method
             this.$store.commit("interactiveMap/addCard", card);
+            // Apply anaerobic conditions if needed
+            if (!jobPrediction.aerobic) {
+              {
+                const payload = {
+                  uuid: card.uuid,
+                  reactionId: "EX_o2_e",
```

Comment:
What about the models where this might have a different identifier?

Reply:
Will create an issue on that